lib/Runtime/Language/AsmJsBuiltInNames.h

```diff
@@ -24,6 +29,9 @@
 #define ASMJS_TYPED_ARRAY_NAMES(name, propertyName) ASMJS_ARRAY_NAMES(name, propertyName)
 #endif
 
+ASMJS_JSBUILTIN_MATH_FUNC_NAMES(Js::PropertyIds::min,   Min     )
+ASMJS_JSBUILTIN_MATH_FUNC_NAMES(Js::PropertyIds::max,   Max     )
```

Comment:
Should probably remove the ASMJS_MATH_FUNC_NAMES for these entires

Reply:
hmm, I guess I removed it. Previously, I was added in the macro #ifdef ENABLE_JS_BUILTINS for those two and #ifndef ENABLE_JS_BUILTINS for the ASMJS_MATH_FUNC_NAMES entries.

Reply:
We actually need the ASMJS_MATH_FUNC_NAMES for those two entries as that is checked elsewhere in the code. I added the two ASMJS_JSBUILTIN_MATH_FUNC_NAMES because asmjs assume calls to math builtins are actually calls to the builtins on the Math object and for these two cases that is not true. So when we compare the entrypoints that should be in the math object, it is wrong for math and min. I added these two so we could compare the actual functions in ASMLink::CheckIsBuiltinFunction.